book/src/super-sql/patterns.md

```diff
@@ -10,14 +10,14 @@ path can be expressed as a [text entity](#text-entity) and need not be quoted:
 from file.json | ...
 ```
 
-Likewise, in the search operator, the syntax for a [regular expression](#regular-expression)
+Likewise, in the [`search` operator](operators/search.md), the syntax for a [regular expression](#regular-expression)
 search can be specified as
 ```
 search /\w+(foo|bar)/
 ```
 instead of the `regexp` function call required in expression context
 ```
-where regexp(r'\w+(foo|bar)')
+where len(regexp(r'\w+(foo|bar)', this)) > 0
```

Comment:
The side-by-side `search` and `where` examples were originally introduced as if they'd produce similar results, but the `where` one didn't work because `regexp` returns either a `null` or an array of matches rather than the boolean that `where` expects, so I've rewritten this `where` example.